templates/error.html

```diff
@@ -42,23 +42,51 @@
   <meta name="theme-color" content="#059184">
 
   <!-- Styles -->
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.3/css/materialize.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.7/css/materialize.min.css">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <link rel="stylesheet" href="{{ cssSource }}">
 </head>
 
 <body>
-  <div class="error-page-header">
-    <a href="/" class="logo-container">
-      <img src="/static/img/logo.png" class="logo" alt="logo" />
-    </a>
-  </div>
-  <div class="error-page-content">
+  <nav>
+    <div class="header">
+      <a href="/" class="logo-container">
+        <img src="/static/img/logo.png" class="logo" alt="logo" />
+      </a>
+      <ul class="right desktop-nav hide-on-med-and-down">
+        <li><a href="/">Home</a></li>
+        <li><a href="/app/search">Search</a></li>
+        <li><a href="/contributors">Contributors</a></li>
+        <li><a href="/contact">Contact</a></li>
+      </ul>
+      <ul class="side-nav" id="mobile">
+        <li><a href="/">Home</a></li>
+        <li><a href="/app">Search</a></li>
```

Comment:
Change this link to be "/app/search", and also please fix it in the Header component as well